### DISCUSSION TITLE
开启cssVar后，我能用cssVar的方式获取根据主题色产生的梯度变量吗？Can I use the CSS Variables (cssVar) method to access the gradient variables generated based on the colorPrimary?

### DISCUSSION THREAD

**Discussion by hengyanqwe** [ant-design/ant-design#50528] [Q&A] [answered]
开启cssVar后，我能用cssVar的方式获取根据主题色产生的梯度变量吗？
Can I use the CSS Variables (cssVar) method to access the gradient variables generated based on the colorPrimary?
![image](https://github.com/user-attachments/assets/56416640-ac0b-48a0-8042-107d26978a10)

**Comment by zombieJ** [comment, marked as answer]
推荐是通过 `theme.useToken` 自己放到 css var 里，不直接使用 antd 提供的 css var 变量。那个 antd 生成的临时变量名是不保证版本一致的。

**Reply by hengyanqwe** [reply]
有参考链接吗？
